tests/__sf_section_time.test.fish

```diff
@@ -3,7 +3,16 @@ source $DIRNAME/mock.fish
 
 function setup
 	spacefish_test_setup
-	set -g SPACEFISH_TIME_TESTING_FLAG true
+
+	function date -a time_format
+		command date --version >/dev/null
```

Comment:
Looks like the Travis logs on macos are complaining about `--version`.
I guess we'll want to redirect both stdout and stderr.
```fish
command date --version >/dev/null ^/dev/null
```